### PR TITLE
Upgrade circle from 1 to 2

### PR DESCRIPTION
**Pull request by SamHerbert**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+experimental:
+  notify:
+    branches:
+      only:
+        - master
+
+jobs:
+  build:
+    docker:
+      - image: managedbyq/circleci-build-images:node8.4.0
```
Comment (mattbriancon) [inline .circleci/config.yml]: Is there anything that image that we really need? Could we use the stock node image instead? The `stretch` or `slim` images might be better still. Also what about updating to 8.11.1?

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    steps:
+      - checkout
+      - run: .circleci/tag_release.sh
+      - run: npm run deploy:npm
+      - run: npm run deploy:docs
+
+workflows:
+  version: 2
+  juice-workflow:
```
Comment (mattbriancon) [inline .circleci/config.yml]: nit: unless it needs a separate name, I've been naming all of the workflows "workflow"

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  build:
+    docker:
+      - image: managedbyq/circleci-build-images:node8.4.0
+    steps:
+      - checkout
+      - run: .circleci/npm_login.sh
+      - run: npm install
+      - run: npm run lint
+
+  deployment:
```
Comment (mattbriancon) [inline .circleci/config.yml]: nit: convention has been to use "deploy" instead of "deployment"

Comment (mattbriancon) [reply .circleci/config.yml]: That said, I see OMD and PD use deployment. Most of the other services use deploy. Up to you.

File: .circleci/config.yml (hunk 1 above)
Comment (SamHerbert) [reply .circleci/config.yml]: Updated to use `8.11.1-slim` -- There was an issue with logging into NPM on 8.11, so I updated the flow to use an NPM token... which I think is an improvement.

**Pull request merged**